### PR TITLE
Update /blog/topics/juju hero section

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Update hero section on `/blog/topics/juju`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/blog/topics/juju
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1889)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1890

### CODE REVIEW

**Comment by anasereijo** [comment]
text should be `h4` as per design and all the other pages.

**Comment by anasereijo** [comment]
@solazio 👍

File: templates/blog/topics/juju.html [hunk 1]
```diff
@@ -6,27 +6,15 @@
...
-          class="u-hide--small u-image-position--bottom"
-          style="padding-top: 2.5rem; max-height: 100%;"
-          alt="" />
-      </div>
+  <section class="p-strip--suru-blog-header">
+    <div class="u-fixed-width">
+      <p>
+        <img class="p-blog-heading__icon" src="https://assets.ubuntu.com/v1/67057b2c-product-page-juju-logo.svg" alt="" width="140" height="50" style="height: 50px;"/>
+      </p>
+      <h2 class="p-heading--four">All the latest news on Juju &ndash; the open source application<br class="u-hide u-show--large" />modelling tool for public and private clouds.</h2>
```
Comment (pmahnke) [inline templates/blog/topics/juju.html]: you need a space before the br

image
```suggestion
      <h2 class="p-heading--four">All the latest news on Juju &ndash; the open source application <br class="u-hide u-show--large" />modelling tool for public and private clouds.</h2>
```

File: templates/blog/topics/juju.html [hunk 2]
```diff
@@ -6,27 +6,15 @@
...
+      <p>
+        <a class="p-link--external p-link--inverted" href="https://jujucharms.com">Learn more about Juju</a>
```
Comment (pmahnke) [inline templates/blog/topics/juju.html]: I think this should go to jaas.ai now
```suggestion
        <a class="p-link--external p-link--inverted" href="https://jaas.ai">Learn more about Juju</a>
```

**Pull request merged**